docs/excel/custom-functions-debugging.md

```diff
@@ -15,7 +15,7 @@ Debugging for custom functions can be accomplished by multiple means, depending
 
 1. Open your custom functions root project folder in VS Code.
 2. You may be prompted to install recommended extensions if you've based this project off of the latest yo office generator project. Select **install all**, which will install [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) and [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extensions for VS Code. If you're not prompted to install these, install these extensions to VS Code manually.
```

Comment:
No need to mention whether it was created from Yo Office. I think this could be rephrased. "When the project is opened, a notification will appear if the recommended extensions are not installed. Click "Install All" to install them." I am not sure that we need to documented the specific ones here (in case they change in the future).